src/fsharp/service/ServiceParseTreeWalk.fs

```diff
@@ -412,8 +412,8 @@ module public AstTraversal =
                      dive synExpr2 synExpr2.Range traverseSynExpr]
                     |> pick expr
                 | SynExpr.Lazy (synExpr, _range) -> traverseSynExpr synExpr
-                | SynExpr.SequentialOrImplicitYield (_sequencePointInfoForSeq, synExpr, synExpr2, _, _range) 
-                | SynExpr.Sequential (_sequencePointInfoForSeq, _, synExpr, synExpr2, _range) -> 
+                | SynExpr.SequentialOrImplicitYield (_SequencePointInfoForSequential, synExpr, synExpr2, _, _range) 
+                | SynExpr.Sequential (_SequencePointInfoForSequential, _, synExpr, synExpr2, _range) -> 
```

Comment:
It seems a local name is now upper-case, it doesn't look correct style-wise to me.

Reply:
good catch thanks, was a case-insensitive global replace